blog/_posts/2021-01-15-snap-announcement.md

```diff
@@ -0,0 +1,158 @@
+However to make proper use of this functionality, it helps to understand snap aliases.
+
+The `root` command itself is an alias, because the snap package is called `root-framework`. When installed from the Snap Store, an alias is created automatically between `root` and `root-framework`.
+For the other binaries such as `hadd`, the original names are namespaced, so hadd's original name is `root-framework.hadd`.
+When you install extra instances of a snap, you must decide which aliases you wish to use manually. 
+
+
+```
+sudo snap set system experimental.parallel-instances=true
+sudo snap install root-framework
+sudo snap install root-framework_nightly --edge --unaliased
+```
+
+Whilst `root` will point to the stable version, you can run `sudo snap prefer root-framework_nightly` so that the next invocation of `root` will be from the nightly branch.
+You can also alias individual commands, or simply use the unaliased binary names, such as `root-framework_nightly.root`.
+
+It's preferable to use snap aliases rather than Bash aliases because they work by placing binaries in the $PATH, for all system users.
+
+The additional installations of snaps will all have their own unique $HOME values, so can have differing `rootlogin.c` files for example.
+
+### Windows Subsystem for Linux 2.
+[https://discourse.ubuntu.com/t/using-snapd-in-wsl2/12113](https://discourse.ubuntu.com/t/using-snapd-in-wsl2/12113)
+
+Officially, running snaps on WSL2 is unsupported.
+This is because WSL2 has a custom init system rather than systemd.
+Unofficially, it's possible to get it working reasonably well anyway, but this is not directly supported.
+
+Having personally tried it, it's possible to get JupyROOT in the snap running in a web browser on Windows, and for some people, the snap on WSL2 might make sense.
+
+Other virtual machine platforms, such as Virtualbox should be able to install Snaps fine.
+
+
+### CUDA & OpenGL
+At the moment, CUDA is not supported, but this may change in the future.
+
+OpenGL generally works fine in a Snap environment for most GPU's.
+A notable exception is amdgpu-pro drivers, and NVidia drivers on Debian and Debian derivatives (but excluding Ubuntu and Ubuntu derivatives).
+These issues can be resolved pending upstream work in the future.
+
+### Performance
+There shouldn't be any observable performance difference from the snap version of ROOT and any other version, a macro that takes an hour to run outside the snap should take an hour to run inside it.
+
+### Creating Independent Executables
+Creating independent executables isn't supported in the snap environment.
+The ABI isn't stable, the compiler toolchain will be foreign to most systems, and the automatic updates would ruin this regularly even if you manage to hack it into working.
+If this is essential to your workflow, you'll be better served with alternate packages.
```

Comment:
alternate -> alternative? (unless it's a technical Snap term)

Reply:
I've made this change